quarkus/config-api/src/main/java/org/keycloak/config/CachingOptions.java

```diff
@@ -40,4 +53,83 @@ public enum Stack {
                     + "The configuration file is relative to the 'conf/' directory.")
             .buildTime(true)
             .build();
+
+    public static final Option<Boolean> JGROUPS_TLS = new OptionBuilder<>(JGROUPS_TLS_ENABLED_PROPERTY, Boolean.class)
+            .category(OptionCategory.CACHE)
+            .description("Encrypts the network communication between Keycloak servers.")
+            .defaultValue(Boolean.FALSE)
+            .buildTime(true)
+            .build();
+
+    public static final Option<String> JGROUPS_TLS_PROTOCOL = new OptionBuilder<>(JGROUPS_TLS_PROTOCOL_PROPERTY, String.class)
+            .category(OptionCategory.CACHE)
+            .description("The TLS protocol version to use. Currently, only TLS v1.2 and v1.3 are supported.")
+            .defaultValue("TLSv1.3")
+            .expectedValues("TLSv1.2", "TLSv1.3")
+            .buildTime(true)
+            .build();
+
+    public static final Option<String> JGROUPS_TLS_PROVIDER = new OptionBuilder<>(JGROUPS_TLS_PROVIDER_PROPERTY, String.class)
```

Comment:
Do you know that people actually do / need to do this in their environment? If it is not a common thing to do, I'd suggest removing this parameter.

Reply:
We don't allow custom providers in Infinispan Server but we have the option to use the JDK provider or the OpenSSL provider (via WildFly-OpenSSL). I'm going to remove this option.